aiida_vasp/workchains/converge.py

```diff
@@ -787,7 +787,14 @@ def results_pw_conv_calc(self):
             self.ctx.converge.run_pw_conv_calcs = False
 
         pwcutoff = self.ctx.converge.settings.pwcutoff
+        total_energy = 0.0
+        max_force = 0.0
+        # Aiida cannot do VBM, yet, so set to zero for now
+        max_valence_band = 0.0
+        gap = 0.0
```

Comment:
Can we figure out another way to do this as some of these values are indeed valid. E.g. a band gap that is zero. Maybe we should consider moving the content to lists and just use `None`? It now dawned on me that this was the reason why I set the `np.nan` in case anyone wonders for future reference.

Reply:
I cannot come up with another way...

> Maybe we should consider moving the content to lists and just use None?

I agree. Besides, I think the current `pw_data` and `k_data` are not so readable for users unless they inspect the implementation. Separating these contents will also be helpful.

Reply:
Yes, I do not really see any need for this having to be numpy, except for consistency. But here, there is an actual case of having the need to use both `float` and `None` in the same container. Can you update with lists?